integration/use-cases/lib/utils.js

```diff
@@ -38,12 +38,15 @@ module.exports = {
   },
   login: async (page, isOIDC, uri, token, username, password) => {
     await page.goto(getUrl(uri));
-    if (isOIDC === true || isOIDC === "true") {
+    if (isOIDC === "true") {
       await page.waitForNavigation();
       await expect(page).toClick("cds-button", {
         text: "Login via OIDC Provider",
       });
-      await page.waitForNavigation();
+      await page.waitForNavigation({ waitUntil: "domcontentloaded" });
+      await expect(page).toMatchElement(".dex-container button", {
+        text: "Log in with Email",
+      });
```

Comment:
will this fix #2653 ?

Reply:
I've triggered the CI (https://app.circleci.com/pipelines/github/antgamdia/kubeapps?branch=minor-ci-fix) and it seems it's working. I'm not 100% sure, but it should bring more stability. Let's see...